.gitignore

```diff
@@ -2,3 +2,4 @@
 *.tar.gz
 *.zip
 *.png
+pacstall-programs/*
```

Comment:
..?

Reply:
I want to exclude my venv for installing pre-commit from pip installation, but @oklopfer say it's make some trouble in future. I deleted it but this PR can't update it automatically